### ISSUE TITLE
Repo and production source code didn't match 

### ISSUE THREAD

**Issue by Drakkarrr** [idurar/idurar-erp-crm#1060]
On demo, I've noticed there's report and register module. Upon cloning the latest version of the repo there is no register route, report page and some other features that demo is showing.

**Comment by Loujayna1** [comment]
Yes, thank you for reporting this issue, it's true the demo provided in the official website and the one you get upon cloning the latest version there is some difference. In addition to missing report and register route, there is also a missing  Branch section. 
There is also no Currency search bar  and no icon for the customization in the navbar of website demo. 
![Screenshot 2024-04-13 224433](https://github.com/idurar/idurar-erp-crm/assets/146457779/545f9812-6d16-40f2-ba67-93798b4508be)
![kkk](https://github.com/idurar/idurar-erp-crm/assets/146457779/2270bdb2-a1c4-4c25-a1ac-85eda3172288)

**Comment by KashifKhan456** [comment]
Can You please help to run this project ???

**Comment by MSNoorul** [comment]
follow the guidelines in readme then use this 
username : admin@demo.com password : admin123
